modules/statics/src/coins.ts

```diff
@@ -2780,6 +2780,14 @@ export const coins = CoinMap.fromCoins([
     '0x6bb04085922d08d1c7de0cfe95f7626a6f54be95',
     UnderlyingAsset['arbeth:eqtyx']
   ),
+  arbethErc20(
+    'd915f6a3-f37e-4da1-b38e-c8c08cae8365',
+    'arbeth:anime',
+    'Animecoin',
+    18,
+    '0x6bb04085922d08d1c7de0cfe95f7626a6f54be95',
```

Comment:
```suggestion
    '0x37a645648df29205c6261289983fb04ecd70b4b3',
```
Contract address for this token should be 0x37a645648df29205c6261289983fb04ecd70b4b3 as per this: https://arbiscan.io/token/0x37a645648df29205c6261289983fb04ecd70b4b3